src/main/public/js/analytics.js

```diff
@@ -13,7 +13,8 @@ xhttp.onreadystatechange = function() {
     })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
     var cookiesPolicy = getCookie("cookies_policy");
-    if (cookiesPolicy.split(',')[1].split(':')[1] === 'true') {
+
```

Comment:
isn't this code in the same case that the file above? I mean with the risk of null/undefined this.response

Reply:
I'm not sure what you are saying here, however the premise for this to occur has changed and this would only resolve if the conditions for the premise has been satisfied.